docs/common/SERVICE-ACCOUNT.md

```diff
@@ -0,0 +1,45 @@
+# Service account
+In order to run Forseti Security, you must add a service account
+to your **organization-level** IAM policy with at least
+the `Browser` role. This allows Forseti Security to perform
```

Comment:
nit: on the org-level permission, they will need iam/securityReviewer (for org resources), and project/editor (for enforcement).  Please feel free to make this in a later change as I realize this PR is more about restructuring.